### PR TITLE
Feature/lobby playerlist

### PR DESCRIPTION
**Pull request by gauthierboeshertz**
Make the lobby player list use a list vie instead of updating a string and use a strings from resources to update the text views

### CODE REVIEW

File: app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java [hunk 1]
```diff
@@ -3,43 +3,88 @@
...
+        String default_player_missing = getString(R.string.lobby_player_missing, missingPlayers);
+        playerMissingView.setText(default_player_missing);
         gameId = (String) getIntent().getStringExtra (getResources().getString(R.string.join_game_lobby_intent_extra_id));
         user = (Player) getIntent().getSerializableExtra(getResources().getString(R.string.join_game_lobby_intent_extra_user));
         runFunctionalities();
+
+    }
+
+
+    public void addQuitLobbyButton(){
```
Comment (Alvaro2112) [inline app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java:56]: Isn't there already a leave lobby button? And when you go back to the Menu you need to pass the player as an extra.

Comment (gauthierboeshertz) [reply app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java:56]: There is one adter the addQuitLobbyButton function call not before?

**Review by Alvaro2112 — APPROVED** [review]

File: app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java [hunk 2]
```diff
@@ -185,6 +202,28 @@ public void playersMissingUpdatesWithDB(){
     }
 
 
+    @Test
```
Comment (Warhedgehog) [inline app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java:205]: How is the fact that the addPlayer fails checked in this test ?

**Review by Warhedgehog — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java (hunk 2 above)
Comment (gauthierboeshertz) [reply app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java:205]: with the exception.except

Comment (Warhedgehog) [reply app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java:205]: Oh sorry missed that

**Review by Warhedgehog — APPROVED** [review]

**Pull request merged**